reader/views.py

```diff
@@ -2730,11 +2730,11 @@ def name_api(request, name):
     name = name[1:] if topic_override else name
     # Number of results to return.  0 indicates no limit
     LIMIT = int(request.GET.get("limit", 10))
-    type = request.GET.get("type", None)
-    active_module = request.GET.get('active_module', None)
+    types = request.GET.getlist("type", None)
```

Comment:
isnt it supposed to be types?
```suggestion
    types = request.GET.getlist("types", None)
```

Reply:
no, the api param is `type` (as it's for one type), but it can show multiple times so the list named `types`